test/SimpleStore.ts

```diff
@@ -65,16 +65,47 @@ export class SimpleStore extends StoreBase {
         return this._subscribeWithEnumKeyData[TriggerKeys.First] + this._subscribeWithEnumKeyData[TriggerKeys.Second];
     }
 
+    @autoSubscribeWithKey(TriggerKeys.First)
+    getSingleKeySingleASKey(@key id: string): number {
+        return this._get(id) + this._subscribeWithEnumKeyData[TriggerKeys.First];
+    }
+
+    @autoSubscribeWithKey([TriggerKeys.First, TriggerKeys.Second])
+    getSingleKeyMultiASKey(@key id: string): number {
+        return this._get(id) + this._subscribeWithEnumKeyData[TriggerKeys.First] + this._subscribeWithEnumKeyData[TriggerKeys.Second];
+    }
+
+    @autoSubscribeWithKey(TriggerKeys.First)
+    getMultiKeySingleASKey(@key id: string, @key id2: string): number {
+        return this._get(id) + this._get(id2) + this._subscribeWithEnumKeyData[TriggerKeys.First];
+    }
+
+    @autoSubscribeWithKey([TriggerKeys.First, TriggerKeys.Second])
+    getMultiKeyMultiASKey(@key id: string, @key id2: string): number {
+        return this._get(id) + this._get(id2) +
+            this._subscribeWithEnumKeyData[TriggerKeys.First] + this._subscribeWithEnumKeyData[TriggerKeys.Second];
+    }
+
     setStoreDataForKeyedSubscription(key: 'A'|'B', data: number): void {
         this._subscribeWithKeyData[key] = data;
         this.trigger(key);
     }
 
+    setStoreDataForCompoundEnumKeyedSubscription(idOrIds: string[], key: TriggerKeys, data: number): void {
+        this._subscribeWithEnumKeyData[key] = data;
+        this.trigger(formCompoundKey(...idOrIds, key));
+    }
+
     setStoreDataForEnumKeyedSubscription(key: TriggerKeys, data: number): void {
         this._subscribeWithEnumKeyData[key] = data;
         this.trigger(key);
     }
 
+    @disableWarnings
+    triggerArbitraryKey(key: string | number | undefined): void {
```

Comment:
Is this needed? Usually a setter will cause the `trigger` as needed. That pattern was used in most/all the places this is currently used.

Reply:
I use it separately to trigger bad keys